_includes/tag_list.html

```diff
@@ -1,4 +1,10 @@
 <div id="sidebar">
+  <div id="advertisement">
+    <a href="https://fastruby.io" target="_blank">
```

Comment:
@ercohen14 Please add the proper utm parameters. @lubc Could you confirm what utm parameters we should use? 

<img width="310" alt="screen shot 2018-10-05 at 4 01 39 pm" src="https://user-images.githubusercontent.com/17584/46557645-968a8c00-c8b8-11e8-968f-0ac42fd748bd.png">

That way we can see a more detailed report in GA ^

Reply:
@ercohen14 @etagwerker Something like `https://fastruby.io/?utm_source=blog&utm_medium=banner&utm_content=sidebar` should work for this.